vnext/ReactCommon/TEMP_UntilReactCommonUpdate/react/renderer/mapbuffer/MapBufferBuilder.cpp

```diff
@@ -0,0 +1,182 @@
+/*
```

Comment:
What's with this TEMP_UntilReactCommonUpdate folder?

Reply:
Yeah, if we need to fork files in core can we do a better job of putting comments in the forked file about what we changed.  -- I would also expect to see an entry in overrides.json for this file -- preferably with a reference to an issue tracking removing the fork.